lib/PkgConfig.pm

```diff
@@ -1086,7 +1086,7 @@ use warnings;
 use Getopt::Long qw(:config no_ignore_case);
 use Pod::Usage;
 
-my $quiet_errors = 1;
+my $print_errors = 1;
```

Comment:
I think the default should be 0?

```
❌ doublethink% pkg-config --exists foobar --print-errors
Package foobar was not found in the pkg-config search path.
Perhaps you should add the directory containing `foobar.pc'
to the PKG_CONFIG_PATH environment variable
Package 'foobar', required by 'virtual:world', not found
❌ doublethink% 
```

Reply:
Thanks for spotting this.  Change made and pushed.